contact/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: contact
 description: TMail Contact extension
-publish_to: 'none'
+publish_to: "none"
```

Comment:
why do we need change it?

Reply:
Reverted

Reply:
Looks like you forgot to revert it. @tddang-linagora